packages/core/src/common/_color-aliases.scss

```diff
@@ -25,6 +25,11 @@ $pt-app-elevated-background-color: $light-gray4 !default;
 $pt-dark-app-elevated-background-color: $dark-gray3 !default;
 
 $pt-outline-color: rgba($blue3, 0.6) !default;
+// 0.752 opacity to meet WCCAG 2.2 minimum contrast guidelines with all light-gray backgrounds in light theme
+// and all dark-gray backgrounds in dark theme
+$pt-focus-indicator-color: rgba($blue2, 0.752) !default;
+// dark theme opacity could be less and still meet minimum contrast, but keep 0.752 to stay consistent
+$pt-dark-focus-indicator-color: rgba($blue5, 0.752) !default;
```

Comment:
$blue5 on $black - [5.77:1 contrast](https://webaim.org/resources/contrastchecker/?fcolor=8ABBFFc0&bcolor=111418)
$blue5 on $dark-gray5 - [3.39:1 contrast](https://webaim.org/resources/contrastchecker/?fcolor=8ABBFFC0&bcolor=404854)

---

for comparison - $blue4 on $dark-gray5 - [2.24:1 contrast](https://webaim.org/resources/contrastchecker/?fcolor=4C90F0C0&bcolor=404854)